javascript/app.js

```diff
@@ -132,8 +134,15 @@ cocktailApp.selectorFadeOut = function (element, interval) {
         element.disabled = true;
     }, interval);
 };
+// method to hide recipe box: 
```

Comment:
method to hide the recipe box since this was done multiple times

Reply:
good call!